bazel/go_repositories.bzl

```diff
@@ -2560,13 +2560,6 @@ def go_repositories():
         version = "v0.2.1",
     )
 
-    go_repository(
-        name = "com_github_xenking_zipstream",
-        importpath = "github.com/xenking/zipstream",
-        sum = "h1:6LfcpXfxO9kAGi0a+2N5C0ZZ6jyG4XULPiogOM7gJBU=",
-        version = "v1.0.1",
-    )
```

Comment:
Why was this removed? It's currently being referenced by enkit/bestie/server/test_result.go.

Reply:
It wasn't added to go.mod. The steps for adding a dep are:

* Add it to go.mod with: `go get -u $DEP_URL`
* Update `go_repositories.bzl` with: `bazel run //:gazelle_update_repos`

so go.mod is the source-of-truth, and manual edits to go_repositories.bzl will be discarded on the next run.

Added zipstream to go.mod to fix this issue